index.js

```diff
@@ -44,149 +44,162 @@ module.exports = (UserProps) => {
   return {
     postcssPlugin: 'postcss-jit-props',
 
-    async Once (node, {Rule, AtRule}) {
-      let target_selector = ':root'
+    prepare() {
+      const UserPropsCopy = JSON.parse(JSON.stringify(UserProps));
```

Comment:
I want to use `structuredClone` but maybe that is still to new?

Reply:
best to wait then, good call 👍🏻